spec/rails_helper.rb

```diff
@@ -78,7 +78,7 @@
     # with the specs, so continue to use transaction strategy for speed.
     driver_shares_db_connection_with_specs = Capybara.current_driver == :rack_test
 
-    page.driver.browser.manage.window.resize_to(1200, 800)
+    page.driver.browser.manage.window.resize_to(1280, 1024)
```

Comment:
Changed this to fix a failure in spec/features/embedded_touchpoints_spec.rb. I think the modal content is narrower and thus longer now so some of the DOM elements referenced in the test were not visible without scrolling. To avoid adding scrolling statements in the test and to avoid redesigning the form in the test, I made the window bigger. I concede that this can't be the solution every time we change the size of some element but 1280x1024 is a pretty reasonable window size. On the other hand, if we specifically chose the original size for the tests, I can put it back.